source/authentication/custom-jwt.txt

```diff
@@ -328,8 +328,14 @@ There are three values that you need to specify for each metadata field:
        | *name*
 
      - The name or path to a field in the JWT that
-       contains the value for the metadata field. To specify the path to a field in
-       an embedded object, use :manual:`dot notation </core/document/#dot-notation>`.
+       contains the value for the metadata field. 
+       
+       If the field *name* contains 
```

Comment:
hmm i wonder if "field name" will be confusing here because there's a Field Name field right after this one – maybe even something like "if the name of the JWT field..." would make it clearer?
i'm also a little confused about the distinction between the field name vs value here – if i'm not mistaken, the "value" case is when the path value is a path to the field and not the direct name of a field? is it possible to make that a little clearer?

Reply:
@wlihui7 - You are absolutely right. It was a sloppy first draft. Please have another look!